public/env-config.js

```diff
@@ -0,0 +1,13 @@
+window._env_ = {
+  REACT_APP_APPLICATION_NAME: "$npm_package_name",
```

Comment:
Would this work? I've assumed that the `$` syntax is some kind of syntax sugar that's applicable in `.env` files and apparently in Gitlab CI files. I would guess that it does not work in actual `.js` files. One way to circumvent is to import the package object and then find the field.

Reply:
Can be fixed/removed. This was just directly from the original .env -file put here as template. Didn't removed yet as didn't know it the value is really needed or just some thing that came with CRA.

Same applies to REACT_APP_VERSION.